common/resources.yaml

```diff
@@ -23,7 +23,7 @@ groups:
     dtype:
     - name: file_object_id
```

Comment:
@mavaylon1 I just noticed that in the "objects" table, we have a field called "object_id". Can we rename this field "file_object_id" to just "object_id" as well?

Reply:
I like it being explicit, especially since on to_dataframe will have both of those columns.

Reply:
Sounds good.